server/errors.ts

```diff
@@ -208,3 +208,19 @@ export function ClientClosedRequestError(
     id: "client_closed_request",
   });
 }
+
+export function DiscordGuildError(
+  message = "User is not a member of the required server"
+) {
+  return httpErrors(400, message, {
+    id: "discord_guild_error",
+  });
+}
+
+export function DiscordGuildRoleError(
+  message = "User does not have the required role"
```

Comment:
nit: Can the message on this be clarified to specify that it refers to discord roles, rather than outline roles/groups?

Reply:
Changed to `User does not have the required role from the Discord server`